src/pages/Admin/Charity/index.tsx

```diff
@@ -1,34 +1,56 @@
 import { Navigate, Route, Routes } from "react-router-dom";
+import { LinkGroup } from "../Sidebar/types";
 import { adminRoutes } from "constants/routes";
 import Layout from "../Layout";
 import Proposal from "../Proposal";
 import Proposals from "../Proposals";
+import { LINKS } from "../constants";
+import Contributions from "./Contributions";
 import Dashboard from "./Dashboard";
 import EditProfile from "./EditProfile";
 import Invest from "./Invest";
-import Nav from "./Nav";
 import Settings from "./Settings";
 import StrategyEditor from "./Settings/StrategyEditor";
 import Templates from "./Templates";
 import WidgetConfigurer from "./WidgetConfigurer";
 import Withdraws from "./Withdraws";
 import { settings } from "./routes";
 
+const LINK_GROUPS: LinkGroup[] = [
```

Comment:
is it possible for type `LinkGroup` to be

```ts
export type LinkGroup = {
  title?: string;
  links: SIDEBAR_ROUTES;
};

so `<Layout/>` could just accept
```

```ts
links: [adminRoutes.index, adminRoutes.withdraws, adminRoutes.contributions]
```

Reply:
The only benefit of this approach would be that less data is copied when creating the `LINK_GROUPS` constants due to current assignment logic (e.g. `links: [LINKS.proposals]` copies the `LINKS.proposals` object into the `links` array). The drawback would be that now `Sidebar` would have `LINK_GROUPS` as a dependency, which means it would be tightly related to admin-related routes and the corresponding data. 
The benefit is not worth the drawback IMO, as the data-copying is not performance-intensive (only a handful of very small objects are copied) and is done only once, while leaving the `Sidebar` potentially re-usable.

Am I missing some other benefit of the suggested approach?

Reply:
EDIT: `Sidebar` is already dependent on the admin pages somewhat due to its `Header` component depending on `useAdminResources`, but the benefit of making an additional dependency is not that significant IMO.